File: OrnaTrack/Stock/UserControl/ItemsUserControl.vb
Public Class ItemsUserControl
    Private _StockID As Integer
    Public Property CurrentItem As Item
    Public Property StockID As Integer
        Get
            Return _StockID
        End Get
        Set
            _StockID = Value

            Dim dr As OleDb.OleDbDataReader = DataReader($"Select img_path From Stock_Data Where ID={StockID}")
            While dr.Read
                If Not IsDBNull(dr(0)) Then
                    ImageBox.ImageLocation = dr(0)
                End If
            End While

        End Set
    End Property

    Private Sub DetailsButton_Click(sender As Object, e As EventArgs) Handles DetailsButton.Click

    End Sub

End Class

File: FirstApp/Stock/UserControl/ItemsUserControl.cs
using System;
using Microsoft.VisualBasic.CompilerServices;

namespace FirstApp
{
    public partial class ItemsUserControl
    {
        private int _StockID;
        public Item CurrentItem { get; set; }
        public int StockID
        {
            get
            {
                return _StockID;
            }
            set
            {
                _StockID = value;

                System.Data.OleDb.OleDbDataReader dr = (System.Data.OleDb.OleDbDataReader)ProjectModule.DataReader($"Select img_path From Stock_Data Where ID={StockID}");
                while (dr.Read())
                {
                    if (!(dr[0] is DBNull))
                    {
                        ImageBox.ImageLocation = Conversions.ToString(dr[0]);
                    }
                }

            }
        }

        public ItemsUserControl()
        {
            InitializeComponent();
            DetailsButton = _DetailsButton;
            _DetailsButton.Name = "DetailsButton";
        }

        private void DetailsButton_Click(object sender, EventArgs e)
        {

        }

    }
}